News.md

```diff
@@ -0,0 +1,5 @@
+# DataFramesMeta v0.6 Release Notes
+
+* The order of rows after a `@transform(gd::GroupedDataFrame, args...)` now matches the 
+order of rows returned after `DataFrames.transform(gd::GroupedDataFrame, args...)`. 
+`@select` now supports `GroupedDataFrame` ([#180])
```

Comment:
```suggestion
  order of rows returned after `DataFrames.transform(gd::GroupedDataFrame, args...)`. 
  `@select` now supports `GroupedDataFrame` ([#180])
```
(I think)

Reply:
Added a new bullet for `@select`, indentation as requested.